src/pages/verantwoording.tsx

```diff
@@ -68,16 +66,20 @@ const Verantwoording: FCWithLayout<{ text: any; lastGenerated: string }> = (
             <h2>{text.verantwoording.title}</h2>
             <p>{text.verantwoording.paragraaf}</p>
             <article className={styles.faqList}>
-              {text.verantwoording.cijfers.map((item: ICijfer) => (
-                <Fragment key={`item-${item.cijfer}`}>
-                  <h3>{item.cijfer}</h3>
-                  <div
-                    dangerouslySetInnerHTML={{
-                      __html: item.verantwoording,
-                    }}
-                  />
-                </Fragment>
-              ))}
+              {text.verantwoording.cijfers.map(
+                (item: ICijfer) =>
+                  item.verantwoording &&
+                  item.cijfer && (
+                    <Fragment key={`item-${item.cijfer}`}>
```

Comment:
```suggestion
                    <Fragment key={item.cijfer}>
```
Not your code, since it was already there, but this would have the same effect